modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request_resolution.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequestResolution < ApplicationRecord
```

Comment:
this is where we can define that mixin for use in the sub types like in this earlier sketch [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/19600).